packages/make-styles/src/types.ts

```diff
@@ -50,12 +50,15 @@ type MakeStylesUnsupportedCSSProperties = {
 type MakeStylesCSSProperties = Omit<
   CSS.Properties<MakeStylesCSSValue>,
   // We have custom definition for "animationName"
-  'animationName'
+  'animationName' | 'fontWeight'
 > &
   MakeStylesUnsupportedCSSProperties;
 
 export type MakeStylesStrictCSSObject = MakeStylesCSSProperties &
-  MakeStylesCSSPseudos & { animationName?: MakeStylesAnimation | MakeStylesAnimation[] | CSS.AnimationProperty };
+  MakeStylesCSSPseudos & {
+    animationName?: MakeStylesAnimation | MakeStylesAnimation[] | CSS.AnimationProperty;
+    fontWeight?: CSS.Properties['fontWeight'] | string;
```

Comment:
Oh, I forgot that we use `csstype@2` that does not allow to configure values for numeric properties, in v3 it's done much better:
https://github.com/frenic/csstype/blob/eb8dae253fd1a00301b92187b06596f845d02b20/index.d.ts#L8818

Let's merge this one then for now.